climaf/classes.py

```diff
@@ -23,6 +23,7 @@
 
 #: Dictionary of frequency names dictionaries
 frequencies=dict()
+realms = dict()
```

Comment:
test comment

Reply:
Totally. I really have to go through writing the documentation...